### ISSUE TITLE
BUG | It should be FROM not TABLE

### ISSUE THREAD

**Issue by kingabzpro** [v1a0/sqllex#24]
============== TEMPLATE ============== 

## ✅ DESCRIBE THE BUG 
It should be under FROM in db.select() not TABLE

Example log:
```
 vaccine_db.select(SELECT= ALL, FROM = "Total", LIMIT= 10)

TypeError: select() missing 1 required positional argument: 'TABLE'
...
sqlite3.OperationalError: no such table: PRAGMA_TABLE_INFO
```

## ✅ CODE
Show the code (or just a few lines) raise this bug. Show what methods you call and what the input data is. MAKE SURE IT DOESN'T CONTAINS ANY PRIVATE DATA! 

For example:
```python
vaccine_db.select(SELECT= ALL, FROM = "Total", LIMIT= 10)
```

## SCREENSHOTS
![image](https://user-images.githubusercontent.com/36753484/121801766-70fdd500-cc52-11eb-88a3-7af446b29beb.png)

## ⚠️ CONFIGURATIONS
 - OS: [Linux]
 - Python version: [3.8]
 - Sqllex version: [0.1.9.3]

**Comment by v1a0** [comment]
Hey @kingabzpro, thanks for new issue. 

Few seconds ago I released [sqllex v0.1.9.4](https://pypi.org/project/sqllex/) to fix this issue, please update up to latest version and let me know is this bug absolutely fixed

**Comment by kingabzpro** [comment]
The issues is resolved.
